samples/Samples.Jwt/Controllers/OAuthController.cs

```diff
@@ -0,0 +1,29 @@
+using System.Security.Claims;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GraphQL.Server.Samples.Jwt.Controllers;
+
+public class OAuthController : Controller
+{
+    // sample OAuth2-compatible authorization endpoint supporting the 'client credentials' flow
+    [HttpGet]
+    [HttpPost]
+    [Route("token")]
+    public IActionResult Token(string grant_type, string client_id, string client_secret)
+    {
+        // validate the provided client id and client secret
+        if (grant_type == "client_credentials" && client_id == "sampleClientId" && client_secret == "sampleSecret")
+        {
+            // provide a signed JWT token with an 'Administrator' role claim
+            var token = JwtHelper.Instance.CreateSignedToken(new Claim("role", "Administrator"));
```

Comment:
Notice here the JWT claim type of "role".  Finding documentation describing that "role" is the proper claim type is not easy, as many lists of JWT claim types omit this claim type.  Some list it as "roles", and perhaps that is more correct.  Nevertheless, it is apparent that the official MS JWT implementation honors the claim type of "role" and translates it for use within .NET accordingly.  So that's what I have been using.

Reply:
See: https://www.iana.org/assignments/jwt/jwt.xhtml#claims

We could test "roles" here to see if it works also...

Reply:
As you wish.